packages/core/src/components/Pagination/index.tsx

```diff
@@ -164,16 +177,47 @@ export class Pagination extends React.Component<Props & WithStylesProps> {
             )}
           </Text>
         </div>
-      </Row>
+        <div {...css(styles.next)}>
+          {nextPage}
+          {lastPage}
+        </div>
+      </div>
     );
   }
 }
 
 export default withStyles(
-  () => ({
-    centered: {
-      width: '100%',
-      textAlign: 'center',
+  ({ unit }) => ({
+    wrapper: {
+      display: 'grid',
+      gridTemplateAreas: '"previous page next"',
+      gridTemplateColumns: 'auto 1fr auto',
+      gridColumnGap: unit * 2,
+      alignItems: 'center',
+      justifyItems: 'center',
+    },
+    page: {
```

Comment:
small nit - it's also helpful to add new lines/space between blocks of style. helps with readability and scanning. e.g.:

```js
wrapper: {
  // ..
},

page: {
  // ..
},
```